assets/js/components/ClusterDetails/ClusterInfoBox.test.jsx

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import { render, screen } from '@testing-library/react';
+import '@testing-library/jest-dom';
+import ClusterInfoBox from './ClusterInfoBox';
+
+describe('Provider Label', () => {
+  it('should display AWS as the provider and HA Scenario: "HANA scale-up"', () => {
+    render(<ClusterInfoBox haScenario="hana_scale_up" provider="aws" />);
+    expect(screen.getAllByText(/AWS/)).toBeTruthy();
```

Comment:
We could even test tha the scenario is follwing the `HA scenario` header and the provider text the `Provider` header. Just for completness